File: C/if_else/absolutevalue.c
// absolute value in which we put the mod so that it convert the negative value into positive
// -8 ko 8 
// 7 ko 7
#include<stdio.h>
int main(){
    int n;
    printf("enter the number");
    scanf("%d",&n);
    if(n>=1)
    {
        printf("absolue value:%d",n);
    }
    else{
        n=-1*n;
        printf("absolute value: %d",n);
    }
    return 0;

}
/*
if(n<0){ //if n is negative
 n=n*(-1)
 }
 printf("the absolute value is :%d",n);
 */

File: Java/conditionals/absolutevalue.java
import java.util.Scanner;;
public class absolutevalue {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("enter the number");
        int n=sc.nextInt();
        // absolute value-round off negative wal ko positive ma convert krne hai
        if(n>0){
            System.out.println("the absolue value is :"+n);
        }
        else //if(n<0)
        {
            n=-(n);
            System.out.println("the absolue value is :"+n);
        }
    }
}
